modules/isDir.js

```diff
@@ -0,0 +1,32 @@
+const fs= require("fs");
+const isMarkdown= require("./isMarkdown.js");
```

Comment:
cuida mucho los espacios en blanco, aunque estos no impactan en el desempeño de la aplicación si es de gran ayuda que tengan un buen formato para que sean legibles
```suggestion
const isMarkdown = require("./isMarkdown.js");
```